### PR TITLE
[WIP] add use_cuda_graph arg 

### PR DESCRIPTION
**Pull request by AntonioMirarchi**
Acoording to [#222](https://github.com/torchmd/torchmd-net/pull/222) from torchmd-net, this PR adds a new arg to the external module in order to allow to capture a TorchMD-Net model and to run it in TorchMD

### CODE REVIEW

File: torchmd/run.py [hunk 1]
```diff
@@ -194,8 +194,15 @@ def setup(args):
...
-        output_transform = args.external["output_transform"] if "output_transform" in args.external else None
-        external = externalmodule.External(args.external["file"], embeddings, device, output_transform)
+        # remove from args_external the items that have been already passed to the external module
+        args_external = {
+            key: value
+            for key, value in args_external.items()
+            if key not in ["module", "file", "embedding"]
+        }
+        external = externalmodule.External(
+            args.external["file"], embeddings, device, **args.external
```
Comment (RaulPPelaez) [inline torchmd/run.py:204]: Change to `..., device=device, **args.external)`. I can see some module having device as the last argument, for instance.

**Comment by RaulPPelaez** [comment]
Is this External functionality documented anywhere besides here? https://github.com/torchmd/torchmd/pull/57/files

**Comment by RaulPPelaez** [comment]
This LGTM, I think it should go in as is. @stefdoerr pleasae review.

**Review by stefdoerr — APPROVED** [review]

**Comment by AntonioMirarchi** [comment]
Shall we merge this?  [#222 ](https://github.com/torchmd/torchmd-net/pull/222) has already been merged.

**Comment by RaulPPelaez** [comment]
Please merge this @stefdoerr

**Pull request merged**